packages/celer-web-app/src/ui/components/Map/Map.tsx

```diff
@@ -37,35 +39,39 @@ const mapCanvas = new MapCanvas();
 const IconSize = 32;
 const IconSizeSmall = 24;
 
-const InGameOriginGeoCoord = inGameToGeoCoord(inGameCoord(0,0));
-const DynamicCanvasAnchorGeoCoord = inGameToGeoCoord(inGameCoord(-DynamicCanvasSizeX,-DynamicCanvasSizeZ));
+const InGameOriginGeoCoord = inGameToGeoCoord(inGameCoord(0, 0));
+const DynamicCanvasAnchorGeoCoord = inGameToGeoCoord(inGameCoord(-DynamicCanvasSizeX, -DynamicCanvasSizeZ));
 
-export const Map: React.FC<MapProps> = ({icons, lines, manualCenter}) => {
```

Comment:
The filtering logic for icons and lines should really be in MapFrame. Map is a simple component used by MapFrame and it should just render whatever is passed into it

Reply:
It has been moved. I added a usememo as well to make sure filtering the icons and lines wouldn't cause lag.